src/physics_mg.cpp

```diff
@@ -64,8 +64,13 @@ void sample_reaction(Particle& p)
 
   // Play Russian roulette if survival biasing is turned on
   if (settings::survival_biasing) {
-    if (p.wgt() < settings::weight_cutoff) {
-      russian_roulette(p, settings::weight_survive);
+    // if survival normalization is applicable, use normalized weight cutoff and normalized weight survive
+    if ((settings::source_file || settings::surf_source_read)&&(settings::survival_normalization)) {
+      if (p.wgt() < settings::weight_cutoff*p.wgt0()) {
+        russian_roulette(p, settings::weight_survive*p.wgt0());
+      } 
+    } else if (p.wgt() < settings::weight_cutoff) {
+            russian_roulette(p, settings::weight_survive);
```

Comment:
```suggestion
    if (p.wgt() < settings::weight_cutoff*p.wgt0()) {
      russian_roulette(p, settings::weight_survive*p.wgt0());
```
same comment here as above.

Reply:
fine with removing SSR gating, but I do think it is best to be able to turn this feature off